src/Editor.h

```diff
@@ -14,7 +14,7 @@ class Editor
 {
 public:
     Editor() = default;
-    explicit Editor(std::shared_ptr<Scene> const& scene) : m_open_scene(scene) { }
+    explicit Editor(std::shared_ptr<Scene> const& scene) : m_open_scene(scene) { set_style(); }
```

Comment:
If constructor becomes more complex than just { } we should probably move it to the .cpp files
This will avoid longer build times when changing it later.